src/Makefile.am

```diff
@@ -12,3 +12,11 @@ vinumc_SOURCES = \
 		 dry_flex.l \
 		 eval.c \
 		 vinumc.c
+
+vin2dot_SOURCES = \
+		 ast.c \
+		 ast_bison_helpers.c \
+		 dry_bison.y \
+		 dry_flex.l \
+		 eval.c \
+		 vin2dot.c
```

Comment:
Just a suggestion—I’m not super familiar with all the Makefile conventions and best practices, but maybe it’d be cool to store these common source files in another variable, like `vinum_common_SOURCES` or something?

Reply:
For some context, here is the snippet with the two variables sharing the common sources:

```Makefile
vinumc_SOURCES = \
		 ast.c \
		 ast_bison_helpers.c \
		 dry_bison.y \
		 dry_flex.l \
		 eval.c \
		 vinumc.c

vin2dot_SOURCES = \
		 ast.c \
		 ast_bison_helpers.c \
		 dry_bison.y \
		 dry_flex.l \
		 eval.c \
		 vin2dot.c
```